### PR TITLE
Add testing and stories in Space Web

### PR DESCRIPTION
**Pull request by krishnaacc123**

### CODE REVIEW

File: src/atoms/Space/Space.stories.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  large: 'large',
+};
+
+storiesOf('Space', module)
+  .addParameters({
+    component: Space,
+  })
+  .add('default', () => (
+    <Space margin={[1, 1]} padding={[2, 2]}>
+      <Text _weight="bold" _isUnderlined size={select('Size', sizeOptions, 'medium')} as="span">
```
Comment (deorukhkarchaitanya) [inline src/atoms/Space/Space.stories.js]: Don't use any underscore ```_``` props unless necessary(from design)

Comment (krishnaacc123) [reply src/atoms/Space/Space.stories.js]: But it's a part of Text component

File: src/atoms/Space/Space.web.js [hunk 2]
```diff
@@ -4,11 +4,14 @@ import styled from 'styled-components';
...
 
-const Space = styled(({ margin, padding, children, ...props }) => {
+const Space = styled(({ margin, padding, children, style, ...props }) => {
   if (React.Children.toArray(children).length !== 1) {
     throw new Error('Expected a single child for Space component');
   }
-  return React.cloneElement(children, props);
+  return React.cloneElement(children, {
+    ...props,
+    style: [style, children.props.style].filter(Boolean),
```
Comment (deorukhkarchaitanya) [inline src/atoms/Space/Space.web.js]: React.js does not accept Array as a valid type for ```style``` prop. That is only supported on React Native. I tested this out and it crashes. You can reproduce this by trying to pass inline style to Text in your story.

Fix:

```suggestion
    style: { ...style, ...children.props.style },
```

File: src/atoms/Space/Space.stories.js (hunk 1 above)
Comment (deorukhkarchaitanya) [reply src/atoms/Space/Space.stories.js]: All ```_``` props are called "private props" which are meant to be used only as an exception from design

Comment (dhruvdutt) [reply src/atoms/Space/Space.stories.js]: Private props usage shouldn't be part of stories.

File: src/atoms/Space/Space.stories.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  large: 'large',
+};
+
+storiesOf('Space', module)
+  .addParameters({
+    component: Space,
+  })
+  .add('default', () => (
+    <Space margin={[1, 1]} padding={[2, 2]}>
+      <Text size={select('Size', sizeOptions, 'medium')} as="span">
```
Comment (dhruvdutt) [inline src/atoms/Space/Space.stories.js]: Don't think size knob for Text component is useful for Space stories.
Better have knobs around Space component itself.

File: src/atoms/Space/Space.web.js (hunk 2 above)
Comment (krishnaacc123) [reply src/atoms/Space/Space.web.js]: added

**Comment by krishnaacc123** [comment]
Closing this PR. Had some problem with. New PR: https://github.com/razorpay/blade/pull/73

**Pull request closed without merging**